packages/cli/src/runIOS/runIOS.js

```diff
@@ -82,11 +82,10 @@ function runIOS(_: Array<string>, ctx: ContextT, args: FlagsT) {
     }
     if (devices && devices.length > 0) {
       // $FlowIssue: args.device is defined in this context
-      logger.info(`Could not find device with the name: "${args.device}".`);
-      logger.info('Choose one of the following:');
-      printFoundDevices(devices);
+      logger.error(`Could not find device with the name: "${args.device}".
+Choose one of the following:${printFoundDevices(devices)}`);
```

Comment:
`printFoundDevices` outputs logs by itself, so this is wrong

Reply:
@thymikee I changed it to be a list.
There's another place which the output is a list (I think for Android). I changed to be like this to keep some sort of standard. But LMK if I need to revert.

Reply:
Ok, that's good (name could be different now, but there's too much changes to this PR already :D) Thanks for bearing with me!